kie-wb-webapp-common/src/main/webapp/login.jsp

```diff
@@ -24,7 +26,7 @@
   <link rel="stylesheet" href="styles/forms.css">
   <link rel="stylesheet" href="styles/login-screen.css">
   <link rel="shortcut icon" href="favicon.ico" />
-  <title><i18n:message key="LoginTitle">KIE Drools Workbench</i18n:message></title>
+  <title><i18n:message key="LoginTitle"></i18n:message></title>
```

Comment:
It still does. I removed the default text to make it reusable across modules, these string should not be in use anyway.